lib/simplifyExpression.js

```diff
@@ -67,13 +68,6 @@ function stepThrough(node, debug=false) {
     }
   }
 
-  // Update the last step, in case we returned something simplified the last
-  // time that didn't count as a step.
-  if (steps.length > 0) {
```

Comment:
the idea is nested unary minuses cancel out - that's what it was testing

not sure why it isn't counted as a step though - maybe it happens in flatten?